clang/lib/Format/QualifierAlignmentFixer.cpp

```diff
@@ -268,20 +268,24 @@ const FormatToken *LeftRightQualifierAlignmentFixer::analyzeRight(
   if (isPossibleMacro(TypeToken))
     return Tok;
 
+  const bool IsCpp = Style.isCpp();
+
   // The case `const long long int volatile` -> `long long int const volatile`
   // The case `long const long int volatile` -> `long long int const volatile`
   // The case `long long volatile int const` -> `long long int const volatile`
   // The case `const long long volatile int` -> `long long int const volatile`
-  if (TypeToken->isSimpleTypeSpecifier()) {
+  if (TypeToken->isTypeName(IsCpp)) {
```

Comment:
I'm not a fan of this extra argument proliferation

Reply:
Neither am I, but I don't know of a better way. BTW, `Token::isSimpleTypeSpecifier(const LangOptions &LangOpts)` also has a similar parameter.

Reply:
Thanks fine, we'll live with it..